src/components/users/registration/RegistrationPage.js

```diff
@@ -1,23 +1,43 @@
 import { Link } from 'react-router-dom';
-import { useState } from 'react';
+import { useState, useContext } from 'react';
 import UserForm from './UserForm';
 import userBlankData from '../utils/userHelpers';
 import client from '../../../utils/client';
+import { loggedInUserContext } from '../../../Helper/loggedInUserContext';
 import './style.css';
+import { useNavigate } from 'react-router-dom';
 
 const RegistrationPage = () => {
   const [user, setUser] = useState(userBlankData());
-  const [registerResponse, setRegisterResponse] = useState('');
-  const [errorResponse, setErrorResponse] = useState('');
+  // const [registerResponse, setRegisterResponse] = useState('');
```

Comment:
Lots of commented code in this file, let's remove it.

Reply:
The useeffect was specifically used on just the login response which was not being used so we decided to remove it and so there was no point of keeping a useEffect there.

Reply:
I believe that useEffect was checking if the user currently had a token in their localstorage, and if they did, the app would treat them as logged in. Is that done elsewhere now?

Reply:
nope

Reply:
No, I checked it wasn't being anywhere else